config.yaml

```diff
@@ -7,7 +7,7 @@
 #  - ignite: use footloose with ignite and firecracker to create real VMs using:
 #            the ignite backend only works on linux as it requires KVM.
 #            https://github.com/weaveworks/ignite.
-backend: ignite
+backend: docker
```

Comment:
Will this stop using ignite?

Reply:
the Docker mode by default seems to be the better choice for people to try.

Reply:
Why is this? Is it speed, or setup issues with ignite? Or ignite not working on all architectures?

Reply:
yes,
- Docker mode generally faster than Ignite mode (because of the container's nature), while Ignite provides better isolation + real kernel.
- Ignite not working on mac OSX atm. But people tended to try it first on mac. So basically this is the better choice.

Reply:
I would say we not stop using Ignite, we just make the docker mode default but Ignite support is still 100% supported.

Reply:
wdyt? @dholbach

Reply:
Can we maybe make this a separate issue to revisit the decision about the default backend? I'm happy to move to docker for now, if that works better.

Reply:
reverted in 5bf751f